src/store/selectors.js

```diff
@@ -0,0 +1,9 @@
+import { useSelector } from "react-redux";
+
+export const movieGenreFilterSelector = useSelector(
```

Comment:
it should be function, if it's a selector. Not straight away value
```suggestion
export const movieGenreFilterSelector = () => useSelector(
```

Reply:
resolved